Index: src/setupTests.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/setupTests.js	(revision dc4c03393df8cb225b9331787df485978f14aa0b)
+++ src/setupTests.js	(date 1599915078593)
@@ -4,15 +4,3 @@
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
 
-
-/*
- workaround for error in tests:
- TypeError: MutationObserver is not a constructor
-
- Suggested in Option 3 of Release 7.
- https://github.com/testing-library/dom-testing-library/releases/tag/v7.0.0
-
- Might be removed when react-scripts depend on jest>=25
- */
-import MutationObserver from '@sheerun/mutationobserver-shim'
-window.MutationObserver = MutationObserver
